dfp/create_line_items.py

```diff
@@ -171,5 +171,8 @@ def create_line_item_config(name, order_id, placement_ids, ad_unit_ids, cpm_micr
     else:  # creative type is IN_APP_VIDEO
       line_item_config['videoMaxDuration'] = 15000
       line_item_config['targeting']['requestPlatformTargeting'] = {'targetedRequestPlatforms': ['MOBILE_APP','VIDEO_PLAYER']}
-    
+
+  if setup_type in ('Video', 'ADPOD'):
```

Comment:
Please use existing constants

Reply:
Do we want to have this setting for inapp_video and JW player as well? Please check

Reply:
Done.  

Will check if we want to support this for other setups.

Reply:
Please use constants defined in the constants file

Reply:
Can we please track it on the parent ticket ?